### DISCUSSION TITLE
Dauntless Devices CAN 2.0B

### DISCUSSION THREAD

**Discussion by Hye993** [collin80/SavvyCAN#833] [General]
I've purchased a Dauntless Devices CAN 2.0B https://dauntlessdevices.com/product/usb-to-can-2-0b-interface-board/ which they have designed & manufactured to be compatible with Savvycan. I've downloaded v213 and updated the firmware on the CAN module [Dauntless FW], and have tried connecting to the Vehicle Bus [MY16 VW Jetta] to no luck.

Observations include: Total Frames Captured/Frames Per Second remain at 0, connection now says Not Connected, LED light is blinking @ ~ 2sec intervals.

Open question to all for those using Serial CAN [GVRET,] have you had similar issues?

**Comment by collin80** [comment]
I have no experience with that device in particular but I do have some suggestions:
1. Are you sure you're using the right speed? 250k and 500k are common
2. Are you connecting via the OBDII port? Quite a lot of cars filter this port and only respond to queries, not broadcast all car traffic. Sometimes it is necessary to tap the bus somewhere else in the car to really get the traffic.
